service/ceagle.yaml

```diff
@@ -15,6 +15,9 @@ service:
           command: "true"
           type: "exec"
       daemon:
+        dependencies:
```

Comment:
So, I removed dependencies and just added health and availability in configs as is because now there is no easy way how to specify them dynamically. Even more ceagle support only one endpoint of each search. I would suggest to merge it and figure out how to deal with it.